app/pages/adduser/confirmation/manifest.json

```diff
@@ -1,5 +1,6 @@
 {
   "backLinkText": "Back to dashboard",
-  "title": "{{username}} account added",
+  "backLinkHref": "/organisations/{{organisationId}}",
```

Comment:
We might want to be careful here with the relative URLS and it working in the cluster. Think we just want to go down a level

Reply:
it doesn't work consistently if you do that - I was having problems

Reply:
might need a wider discussion

Reply:
Ok let's discuss tomorrow.